fhir-resource-tooling/src/main/java/com/ibm/cohort/tooling/fhir/ValueSetImporter.java

```diff
@@ -0,0 +1,161 @@
+		boolean isDisplayHelp;
+
+		@Parameter(description = "The list of value set spreadsheets to import", required = true)
+		List<String> spreadsheets;
+	}
+
+	private static class ValueSetArtifact {
+
+		MetadataResource resource;
+		String url;
+		String name;
+		String id;
+	}
+
+	private IGenericClient client;
+	private IParser parser;
+
+	private ValueSetImporter(IGenericClient client) {
+		this.client = client;
+		this.parser = client.getFhirContext().newJsonParser();
+	}
+
+	private void importArtifacts(List<ValueSetArtifact> valueSetArtifacts) {
+
+		for (ValueSetArtifact valueSetArtifact : valueSetArtifacts) {
+			Bundle bundle = client.search().forResource(ValueSet.class).where(ValueSet.URL.matches().value(valueSetArtifact.url))
+					.returnBundle(Bundle.class).execute();
+			if (bundle.getEntry().size() > 0) {
+				valueSetArtifact.id = bundle.getEntryFirstRep().getResource().getIdElement().getIdPart();
+			} else {
+				MethodOutcome outcome = client.create().resource(parser.encodeResourceToString(valueSetArtifact.resource)).execute();
+				if (outcome.getCreated()) {
+					valueSetArtifact.id = outcome.getId().getIdPart();
+				}
+			}
+		}
+	}
+
+	private ValueSetArtifact createArtifact(InputStream is) throws IOException {
+		XSSFWorkbook wb = new XSSFWorkbook(is);
+		XSSFSheet mainSheet = wb.getSheetAt(0);
+		ValueSet valueSet = new ValueSet();
+		for (Row currentRow : mainSheet) {
+			if(currentRow.getCell(0) != null && currentRow.getCell(1) != null) {
+				String value = currentRow.getCell(1).getStringCellValue();
+				switch (currentRow.getCell(0).getStringCellValue()) {
+					case "Value Set Name":
+						valueSet.setName(value);
+						valueSet.setTitle(value);
+						break;
+					case "OID":
+						valueSet.setId(value);
+						String url = "http://cts.nlm.nih.gov/fhir/ValueSet/" + value;
+						valueSet.setUrl(url);
+						break;
+					case "Definition Version":
+						valueSet.setVersion(value);
+						break;
+					default:
+						break;
+				}
+			}
+		}
+		XSSFSheet expansionSheet = wb.getSheetAt(1);
+		ValueSet.ValueSetComposeComponent compose = new ValueSet.ValueSetComposeComponent();
+		List<ValueSet.ConceptSetComponent> composeIncludes = new ArrayList<>();
+		for(Row currentRow : expansionSheet){
+			if (currentRow.getCell(0) != null && currentRow.getCell(0).getCellType().equals(CellType.NUMERIC)){
+				ValueSet.ConceptSetComponent component = new ValueSet.ConceptSetComponent();
+				component.setSystem(currentRow.getCell(2).getStringCellValue());
+				ValueSet.ConceptReferenceComponent concept = new ValueSet.ConceptReferenceComponent();
+				concept.setCode(currentRow.getCell(0).getStringCellValue());
+				concept.setDisplay(currentRow.getCell(1).getStringCellValue());
+				List<ValueSet.ConceptReferenceComponent> concepts = new ArrayList<>();
+				concepts.add(concept);
+				component.setConcept(concepts);
+				composeIncludes.add(component);
+			}
```

Comment:
I think we need to have one element under `include` for each unique code system.
It looks like this code will create a unique `include` element for every row in the spreadsheet.